crates/uv-static/src/env_vars.rs

```diff
@@ -1,117 +1,164 @@
+    /// as the overrides file. Uses space-separated list of files.
     pub const UV_OVERRIDE: &'static str = "UV_OVERRIDE";
 
-    /// Equivalent to the `--link-mode` argument. Specifies link mode for the installation.
+    /// Equivalent to the `--link-mode` command-line argument. If set, uv will use this as
+    /// a link mode.
     pub const UV_LINK_MODE: &'static str = "UV_LINK_MODE";
 
-    /// Equivalent to the `--no-build-isolation` argument. Skips build isolation.
+    /// Equivalent to the `--no-build-isolation` command-line argument. If set, uv will
+    /// skip isolation when building source distributions.
     pub const UV_NO_BUILD_ISOLATION: &'static str = "UV_NO_BUILD_ISOLATION";
 
-    /// Equivalent to the `--custom-compile-command` argument. Overrides the command in `requirements.txt`.
+    /// Equivalent to the `--custom-compile-command` command-line argument.
+    /// Used to override uv in the output header of the `requirements.txt` files generated by
+    /// `uv pip compile`. Intended for use-cases in which `uv pip compile` is called from within a wrapper
+    /// script, to include the name of the wrapper script in the output file.
     pub const UV_CUSTOM_COMPILE_COMMAND: &'static str = "UV_CUSTOM_COMPILE_COMMAND";
 
-    /// Equivalent to the `--keyring-provider` argument. Specifies keyring provider.
+    /// Equivalent to the `--keyring-provider` command-line argument. If set, uv
+    /// will use this value as the keyring provider.
     pub const UV_KEYRING_PROVIDER: &'static str = "UV_KEYRING_PROVIDER";
 
-    /// Equivalent to the `--config-file` argument. Path to configuration file.
+    /// Equivalent to the `--config-file` command-line argument. Expects a path to a
+    /// local `uv.toml` file to use as the configuration file.
     pub const UV_CONFIG_FILE: &'static str = "UV_CONFIG_FILE";
 
-    /// Equivalent to the `--no-config` argument. Prevents reading configuration files.
+    /// Equivalent to the `--no-config` command-line argument. If set, uv will not read
+    /// any configuration files from the current directory, parent directories, or user configuration
+    /// directories.
     pub const UV_NO_CONFIG: &'static str = "UV_NO_CONFIG";
 
-    /// Equivalent to the `--exclude-newer` argument. Excludes newer distributions after a date.
+    /// Equivalent to the `--exclude-newer` command-line argument. If set, uv will
+    /// exclude distributions published after the specified date.
     pub const UV_EXCLUDE_NEWER: &'static str = "UV_EXCLUDE_NEWER";
 
-    /// Equivalent to the `--python-preference` argument. Controls preference for Python versions.
+    /// Equivalent to the `--python-preference` command-line argument. Whether uv
+    /// should prefer system or managed Python versions.
     pub const UV_PYTHON_PREFERENCE: &'static str = "UV_PYTHON_PREFERENCE";
 
-    /// Equivalent to the `--no-python-downloads` argument. Disables Python downloads.
+    /// Equivalent to the
```

Comment:
I'm curious why all these changes? I kind of liked the docs here.

Reply:
It is copied from the original `docs/configuration/enviroment.md` file. I think it's more user friendly for a document intended for end-users, rather than a developer facing docstring.